cohttp-lwt/src/server.ml

```diff
@@ -111,32 +112,32 @@ module Make (IO : S.IO) = struct
                 `Response rsp))
       (fun () -> Body.drain_body body)
 
-  let handle_response ~keep_alive oc res body conn_closed handle_client =
+  type conn_action = Call_conn_closed | Call_conn_closed_and_drain of Body.t
+
+  let handle_response ~keep_alive oc res body _conn_closed handle_client =
```

Comment:
```suggestion
  let handle_response ~keep_alive oc res body handle_client =
```

Reply:
Done in [fixup4](https://github.com/gabrielmoise17/ocaml-cohttp/pull/3/commits/4b6dc15371d36a4e6fdfe9b8962b46c12e1caf62)